### PR TITLE
Fix RuntimeErrors properly

### PR DESCRIPTION
**Pull request by cccs-sgaron**

### CODE REVIEW

File: assemblyline_v4_service/common/base.py [hunk 1]
```diff
@@ -23,6 +23,16 @@
 UPDATES_DIR = os.environ.get('UPDATES_DIR', '/updates')
 PRIVILEGED = os.environ.get('PRIVILEGED', 'false') == 'true'
 
+RECOVERABLE_RE_MSG = ["cannot schedule new futures after interpreter shutdown", "can't register atexit after shutdown"]
+
+
+def is_recoverable_runtime_error(error):
+    for msg in RECOVERABLE_RE_MSG:
```
Comment (cccs-rs) [inline assemblyline_v4_service/common/base.py:30]: nitpick: could be written

if any(msg in str(error) for msg in RECOVERABLE_RE_MSG):
    return True

**Review by cccs-rs — APPROVED** [review]

File: assemblyline_v4_service/common/base.py [hunk 2]
```diff
@@ -23,6 +23,16 @@
 UPDATES_DIR = os.environ.get('UPDATES_DIR', '/updates')
 PRIVILEGED = os.environ.get('PRIVILEGED', 'false') == 'true'
 
+RECOVERABLE_RE_MSG = ["cannot schedule new futures after interpreter shutdown", "can't register atexit after shutdown"]
```
Comment (cccs-kevin) [inline assemblyline_v4_service/common/base.py:26]: can't register atexit after shutdown -> can't register **at exit** after shutdown

Comment (cccs-rs) [reply assemblyline_v4_service/common/base.py:26]: weirdly enough the error is actually: can't register atexit after shutdown

Least from the logs that are recorded.

File: assemblyline_v4_service/common/base.py (hunk 1 above)
Comment (cccs-kevin) [reply assemblyline_v4_service/common/base.py:30]: then nvm!

**Pull request merged**